.github/CODEOWNERS

```diff
@@ -719,10 +719,10 @@
 # ServiceOwners:                                                   @privlinksuppgithub
 
 # PRLabel: %Nginx
-/sdk/nginx/                                                        @spencerofwiti @bangbingsyb
+/sdk/nginx/                                                        @spencerofwiti @bangbingsyb @jamesfan
```

Comment:
Given that this is a management library, we should move the nginx block down to the management section around L951 and follow the pattern there to restrict to the Resource Manager path.

Please be sure to maintain alphabetical ordering of label names when moving.